### ISSUE TITLE
Keyboard navigation: Focus indicator is not present on the close button in settings panel

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-web#717]
## Description

When the settings panel is opened using a keyboard, the focus is placed on the close button by default. But the focus ring is missing here.

**To Reproduce**

1. Go to 'https://www.microsoft.com/en-us'
2. Navigate to the settings button in details view using keybaord
3. Open the settings panel using keyboard
4. Observe that the focus indicator on the close button is missing

## Expected behavior

Display the focus ring around close button

## Screenshots

![image](https://user-images.githubusercontent.com/22068434/58661392-b570df00-82dc-11e9-9056-529364f0bd78.png)

## Extension 

- Accessibility Insights for Web - prod
- Browser Version: Chrome 74.0.3729.169

**Comment by devaradhanm** [comment]
@iamrafan , i see that the focus is on the close button but we don't highlight it on opening the settings dialog. but we do have a focus ring the next time the focus is on the close button . I will check why we have this behaviour

**Comment by devaradhanm** [comment]
office fabric adds a class ms-Fabric--isFocusVisible on body tag, after u start interacting with the dialog. so, the close button only gets focus the second time u focus the element. We can have a workaround fix on our end, but this needs to be fixed on office fabric side.

**Comment by iamrafan** [comment]
@devaradhanm looks like this has been fixed in the latest office fabric update

**Comment by devaradhanm** [comment]
@iamrafan , i still see this issue there as well. (as soon as the panel is open, i dont see the focus, even though the focus is on the close button. (if u hit enter the panel will dismiss). But the second time when u tab to the close button the focus is added.
